clients/admin-ui/src/flags.json

```
@@ -53,5 +53,11 @@
     "development": true,
     "test": true,
     "production": false
+  },
+  "datahub": {
+    "description": "Use Datahub for data discovery and detection",
```

```suggestion
    "description": "Share Fides data categories with your Datahub instance",
```
or something that better reflects what the integration is going to do